### PR TITLE
Update projects versions and readme files.

### PR DESCRIPTION
**Pull request by laura-rodriguez**
Versions:
* `Okta.AspNet.Abstractions` from `3.0.1` -> `3.0.2` - Comments/Doc were added to some public properties
* `Okta.AspNet` from `1.1.6` -> `1.2.0` - Deprecated property / New feature - No breaking changes
* `Okta.AspNet.Core` from `1.1.5` -> `1.2.0` - New features - No breaking changes

This is gonna be merged after #83.

### CODE REVIEW

File: Okta.AspNetCore/Okta.AspNetCore.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Official Okta middleware for ASP.NET Core 2.0+. Easily add authentication and authorization to ASP.NET Core applications.</Description>
     <Copyright>(c) 2018 Okta, Inc.</Copyright>
```
Comment (OkGoDoIt) [inline Okta.AspNetCore/Okta.AspNetCore.csproj]: Maybe we should update this to 2019 like in https://github.com/okta/okta-aspnet/pull/84/files#diff-6dd76ea6f734baad2516d413eed0f382R5

I updated that other one when I updated aspnet to 1.1.6

File: docs/aspnetcore-mvc.md [hunk 2]
```diff
@@ -62,7 +61,9 @@ The `OktaMvcOptions` class configures the Okta middleware. You can see all the a
...
 | PostLogoutRedirectUri     | No           | The location Okta should redirect to after logout. If blank, Okta will redirect to the Okta login page. |
 | Scope                     | No           | The OAuth 2.0/OpenID Connect scopes to request when logging in. The default value is `openid profile`. |
-| GetClaimsFromUserInfoEndpoint | No       | Whether to retrieve additional claims from the UserInfo endpoint after login (not usually necessary). The default value is `false`. |
+| GetClaimsFromUserInfoEndpoint | No       | Whether to retrieve additional claims from the UserInfo endpoint after login (not usually necessary). The default value is `true`. |
 | ClockSkew                 | No           | The clock skew allowed when validating tokens. The default value is 2 minutes. |
+| OnTokenValidated                 | No           | The event invoked after the security token has passed validation and a ClaimsIdentity has been generated. |
+| OnUserInformationReceived                 | No           | The event invoked when user information is retrieved from the UserInfoEndpoint. The `GetClaimsFromUserInfoEndpoint` value must be `true` when using this event. |
```
Comment (OkGoDoIt) [inline docs/aspnetcore-mvc.md]: Do these events have an equivalent in the asp.net framework side?  If so, maybe we want to call them out in that readme for consistency?

**Comment by laura-rodriguez** [comment]
> Feels weird that on the aspnet framework side the GetClaimsFromUserInfoEndpoint went from "not usually necessary" to deprecated, but on the aspnet core side it went from "not usually necessary" to true-by-default and required for one of the events. Is it possible that change might cause issues in any way? (I mean, are there potential side effects of automatically getting claims from user info endpoint that an upgrading user might not be aware of?)
> 
> There's now an asymmetry between the two libraries, which may be unavoidable but it might be good to explain in the readme for the aspnet framework side how to do the equivalent of the events on the core side, or if it's even possible.
> 
> Also I would recommend updating the copyright year in the aspnet.core project to 2019.



> Feels weird that on the aspnet framework side the GetClaimsFromUserInfoEndpoint went from "not usually necessary" to deprecated, but on the aspnet core side it went from "not usually necessary" to true-by-default and required for one of the events. Is it possible that change might cause issues in any way? (I mean, are there potential side effects of automatically getting claims from user info endpoint that an upgrading user might not be aware of?)
> 
> There's now an asymmetry between the two libraries, which may be unavoidable but it might be good to explain in the readme for the aspnet framework side how to do the equivalent of the events on the core side, or if it's even possible.
> 
> Also I would recommend updating the copyright year in the aspnet.core project to 2019.

Good point. The reason why we're getting rid of the `UserInfoEndpoint` in Asp.Net is that it doesn't add additional information. In Asp.Net we are using the hybrid flow (scope = code id_token), and this because of a limitation in project Katana. The hybrid flow makes that we get always the same claims regardless of Okta configuration (In Okta you can configure claims to come as part of id_token or userInfo) as part of the "hybrid" id token. So, getting claims from UserInfo endpoint didn't add any additional value and caused confusion. 
In Asp.Net Core things are different as it's using the code flow. Basically, you get additional information from the UserInfo endpoint (claims marked as "from userInfo" in Okta).
My plan is to add this information in the release so we can keep the readme simple.

Regarding the copyright, you're right. I'll update it soon.

File: docs/aspnetcore-mvc.md (hunk 2 above)
Comment (OkGoDoIt) [reply docs/aspnetcore-mvc.md]: #83 added similar events, if feels like those should be added to its readme as well https://github.com/okta/okta-aspnet/blame/master/docs/aspnet4x-mvc.md#L60

**Pull request merged**